sycl/test-e2e/FilterSelector/select_device.cpp

```diff
@@ -11,7 +11,7 @@
 // Checks that no device is selected when no device of desired type is
 // available.
 //
-// REQUIRES: cpu,gpu,accelerator
+// REQUIRES: any-device-is-cpu, any-device-is-gpu, any-device-is-accelerator
```

Comment:
This should not be an E2E test at all, it  should be moved to unit-tests where we can emulate all environments and check that we query the right device, or just directly check that we parse the env variable correctly, similar to what I did in #15774